installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc

```diff
@@ -5,10 +5,38 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The following troubleshooting sections address Installer Provisioned Infrastructure (IPI) troubleshooting.
 
-// For general, {product-title} troubleshooting, see <xref>.
+.Troubleshooting the installer workflow
 
-include::modules/ipi-install-troubleshooting-the-bootstrap-vm.adoc[leveloffset=+1]
+Prior to troubleshooting the installation environment, it is critical to understand the overall flow of the IPI installation on bare metal. The diagrams below provide a troubleshooting flow with a step-by-step breakdown for the environment.
 
-include::modules/ipi-install-troubleshooting-the-control-plane.adoc[leveloffset=+1]
+image:flow1.png[Flow-Diagram-1]
+
+_Workflow 1 of 4_ illustrates a troubleshooting workflow when the `install-config.yaml` file has errors or the RHCOS images are inaccessible.  Troubleshooting suggestions can be found at <<install_config,Troubleshooting `install-config.yaml`>>.
```

Comment:
Looks like this link is broken. We don't use this link format. It must be in xref: format per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-internal-content